src/Raven.Studio/wwwroot/App/views/wizard/setupShell.html

```diff
@@ -8,12 +8,34 @@ <h2 class="margin-none"> <strong>RavenDB Setup Wizard</strong></h2>
         </div>
         <div class="flex-separator"></div>
         <div>
-            <a target="_blank" data-bind="attr: { href: 'https://ravendb.net/l/8H5IPX/' + $root.constructor.clientVersion() }">
+            
+            <a target="_blank" data-bind="visible: $data.constructor.deploymentEnvironment() === 'Custom', attr: { href: 'https://ravendb.net/l/8H5IPX/' + $root.constructor.clientVersion() }">
                 <div class="label label-info padding padding-sm">
                     <i class="icon-info"></i>
                     <span>Need assistance? Click here for a detailed walk-through and setup examples.</span>
                 </div>
             </a>
+            
+            <a target="_blank" data-bind="visible: $data.constructor.deploymentEnvironment() === 'AwsLinux', attr: { href: 'https://ravendb.net/l/VPQ69V/' + $root.constructor.clientVersion() }">
+                <div class="label label-info padding padding-sm">
+                    <i class="icon-info"></i>
+                    <span>Need assistance? Click here for a detailed instructions for AWS deployment</span>
```

Comment:
remove the 'a'... click here for detailed instructions

Reply:
force pushed my change. Thanks!